components/style/gecko/values.rs

```diff
@@ -414,7 +414,8 @@ impl GeckoStyleCoordConvertible for ScrollSnapPoint<LengthOrPercentage> {
         Some(
             match coord.unit() {
                 nsStyleUnit::eStyleUnit_None => ScrollSnapPoint::None,
-                nsStyleUnit::eStyleUnit_Coord | nsStyleUnit::eStyleUnit_Percent  =>
+                nsStyleUnit::eStyleUnit_Coord | nsStyleUnit::eStyleUnit_Percent |
+                nsStyleUnit::eStyleUnit_Calc  =>
                     ScrollSnapPoint::Repeat(LengthOrPercentage::from_gecko_style_coord(coord)
```

Comment:
BTW, is there any particular reason we can't just do:

`_ => ScrollSnapPoint::Repeat(...)`

Given we're panicking anyway...